crux-test/src/crux/fixtures/api.clj

```diff
@@ -0,0 +1,32 @@
+(ns crux.fixtures.api
```

Comment:
think this already exists as `crux.fixtures`?

Reply:
have deleted - it crept back in